src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue

```diff
@@ -216,7 +216,7 @@ export default {
 	user-select: none;
 	min-height: $clickable-area;
 	border-radius: $clickable-area;
-	padding: 4px $icon-margin;
+	padding: 4px math.div($clickable-area - 24px, 2);
```

Comment:
Where is this 24 coming from?
Ideally this should be all calculated.

Reply:
You are right, it needs to be calculated, as it currently does not work for `type="switch"`. This value is the size of the icon set here:
https://github.com/nextcloud-libraries/nextcloud-vue/blob/dae4eca0f022724619c7b912133094d75c886769/src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue#L498C11-L502

I guess it should look like this in the end (don't mind the background-colors, just set for visualization):
![grafik](https://github.com/nextcloud-libraries/nextcloud-vue/assets/2496460/1b506d31-f561-47de-b1a4-650af3a403ae)

Reply:
Fixed.

Reply:
Updated the screenshot in the initial post.